testing/main_test.go

```diff
@@ -127,7 +127,7 @@ func initORMApp(t *testing.T, app application, dbURL *url.URL) (killFunc, restar
 
 // waitForInit retries until a connection is successfully established.
 func waitForInit() error {
-	const maxWait = 15 * time.Second
+	const maxWait = 3 * time.Minute
```

Comment:
Gradle+the JVM takes a while to start up sometimes (10s of seconds). This is super conservative, but I don't see much of a downside to that.